addons/sale_mrp/sale_mrp.py

```diff
@@ -11,14 +11,15 @@ class MrpProduction(models.Model):
     sale_name = fields.Char(compute='_compute_sale_name_sale_ref', string='Sale Name', help='Indicate the name of sales order.')
     sale_ref = fields.Char(compute='_compute_sale_name_sale_ref', string='Sale Reference', help='Indicate the Customer Reference from sales order.')
 
+    def _get_parent_move(self, move):
+        if move.move_dest_id:
+            return self._get_parent_move(move.move_dest_id)
+        return move
+
     @api.multi
     def _compute_sale_name_sale_ref(self):
-        def get_parent_move(move):
-            if move.move_dest_id:
-                return get_parent_move(move.move_dest_id)
-            return move
         for production in self:
-            move = get_parent_move(production.move_finished_ids[0])
+            move = production._get_parent_move(production.move_finished_ids[0])
```

Comment:
here i need to find recursive move, i mean backorder of backorder , and then set inside chatter link.
so for that required to create new method, but in "_compute_sale_name_sale_ref" method is already there,
so i just used that method keep that method separtor and use it in both . so no need to define new method . for same code..